.gitignore

```diff
@@ -1,4 +1,5 @@
 composer.lock
+.php-version
```

Comment:
What is this for ?

Reply:
Both `phpenv` and the symfony cli utilize this file to use a specific PHP version if multiple versions are installed locally.